audformat/core/table.py

```diff
@@ -1562,6 +1555,26 @@ def _get_by_index(
         return result, result_is_copy
 
 
+def _assert_file_level_dtype_is_string(
```

Comment:
`assert` sounds to me as if it would raise an error if dtype is not `'string'`.

Maybe rename to `_convert_file_level_dtype_to_string()` or `_ensure_file_level_dtype_is_string()`?